src/onnxruntime.cc

```diff
@@ -1353,9 +1397,6 @@ ModelInstanceState::SetInputTensors(
       size_t dst_buffer_byte_size;
       TRITONSERVER_MemoryType dst_memory_type;
       int64_t dst_memory_type_id;
-      std::vector<std::pair<TRITONSERVER_MemoryType, int64_t>>
-          allowed_input_types;
-      allowed_input_types = {{TRITONSERVER_MEMORY_CPU, 0}};
```

Comment:
Batch inputs are always created on the CPU I believe, so you likely want to leave this like it is. @GuanLuo  can you confirm.

Reply:
@GuanLuo can you confirm this?

Reply:
I made the changes with the assumptions that batch inputs are always created on CPU. @GuanLuo will need your confirmation before I can check this in. Thanks!

Reply:
Yes, batch input will be created on CPU as Triton needs to modify the buffer according to the batch input type